<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="interpolateBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Interpolate methods</string>
    </property>
    <widget class="QPushButton" name="linearButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Linear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="timeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
    <widget class="QPushButton" name="indexButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Index</string>
     </property>
    </widget>
    <widget class="QPushButton" name="valuesButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Values</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nearestButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Nearest</string>
     </property>
    </widget>
    <widget class="QPushButton" name="padButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pad</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zeroButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Zero</string>
     </property>
    </widget>
    <widget class="QPushButton" name="slinearButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Slinear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quadraticButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Quadratic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cubicButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cubic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="polynomialButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Polynominal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="kroghButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Krogh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="piePolButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Piecewise Polynomial</string>
     </property>
    </widget>
    <widget class="QPushButton" name="splineButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Spline</string>
     </property>
    </widget>
    <widget class="QPushButton" name="barycentricButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Barycentric</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pchipButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pchip</string>
     </property>
    </widget>
    <widget class="QPushButton" name="akimaButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Akima</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cubicsplineButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cubicspline</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>720</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>511</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>511</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>540</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>How old are you?</string>
    </property>
    <widget class="QSpinBox" name="spinBoxAge">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
    </widget>
    <widget class="QLabel" name="ageLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>You are _ years old</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>620</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Select your gender:</string>
    </property>
    <widget class="QRadioButton" name="maleRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="femaleRadio">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
    <widget class="QLabel" name="genderSelectionLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>You are...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>540</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Graphs</string>
    </property>
    <widget class="QPushButton" name="dat3DButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>3D Graph</string>
     </property>
    </widget>
    <widget class="QPushButton" name="subplButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Subplot Graph</string>
     </property>
    </widget>
    <widget class="QPushButton" name="myltiButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Mylti Graph</string>
     </property>
    </widget>
    <widget class="QPushButton" name="primerButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Primer Graph</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>550</y>
      <width>69</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Primer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mylti</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Subplot</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3D</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="showGraph">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showGraph2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuxdvxdv">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionCloseFile"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFont"/>
    <addaction name="actionColor"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuxdvxdv"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="toolSAVE"/>
   <addaction name="toolPrint"/>
   <addaction name="separator"/>
   <addaction name="toolInterpolate"/>
   <addaction name="separator"/>
   <addaction name="toolCalendar"/>
   <addaction name="separator"/>
   <addaction name="toolExit"/>
  </widget>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open CSV file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save file as</string>
   </property>
  </action>
  <action name="actionCloseFile">
   <property name="text">
    <string>Close CSV file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="toolExit">
   <property name="icon">
    <iconset>
     <normaloff>../2048px-PublicInformationSymbol_EmergencyExit.svg.png</normaloff>../2048px-PublicInformationSymbol_EmergencyExit.svg.png</iconset>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </action>
  <action name="toolSAVE">
   <property name="icon">
    <iconset>
     <normaloff>../84205_save_512x512.png</normaloff>../84205_save_512x512.png</iconset>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="toolPrint">
   <property name="icon">
    <iconset>
     <normaloff>../icon-256x256.png</normaloff>../icon-256x256.png</iconset>
   </property>
   <property name="text">
    <string>PRINT</string>
   </property>
  </action>
  <action name="toolInterpolate">
   <property name="icon">
    <iconset>
     <normaloff>../1200px-LinearInterpolation.svg.png</normaloff>../1200px-LinearInterpolation.svg.png</iconset>
   </property>
   <property name="text">
    <string>Interpolate</string>
   </property>
  </action>
  <action name="toolCalendar">
   <property name="icon">
    <iconset>
     <normaloff>../icon_B716bOJxyK28xpLqkebShllKASPFjHdZiRMjEe0oFxY= (1).png</normaloff>../icon_B716bOJxyK28xpLqkebShllKASPFjHdZiRMjEe0oFxY= (1).png</iconset>
   </property>
   <property name="text">
    <string>calendar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
